roottest/root/io/customStreamer/CMakeLists.txt

```diff
@@ -2,3 +2,10 @@ ROOTTEST_ADD_TEST(run
                   MACRO run.C
                   OUTREF customStreamerTest.ref)
 
+ROOTTEST_GENERATE_DICTIONARY(StreamerThrowClass StreamerThrowClass.h
+                             LINKDEF StreamerThrowClass_LinkDef.h
+                             FIXTURES_SETUP root-io-customStreamer-StreamerThrow)
+ROOTTEST_ADD_TEST(StreamerThrow
+                  MACRO StreamerThrow.C
```

Comment:
```suggestion
                  MACRO StreamerThrow.C+
```

Reply:
in addition you can use a variation of
```
ROOTEXE_OPTS -e ".L StreamerThrowClass"
```
to explicitly load the dictionary library.
Side note, if I recall correct that syntax has challenges to use `;`, so for 'quiet' C++ statement we used `-e "(void)somefunction()"`

Furthermore, the ACLiC compilation can be done separately by adding something like:
```
ROOTTEST_COMPILE_MACRO(StreamerThrow.C
                        BUILDLIB StreamerThrowClass
                        FIXTURES_REQUIRED root-io-customStreamer-StreamerThrowLib
                        FIXTURES_SETUP  root-io-customStreamer-StreamerThrowACLiC)
```